### PR TITLE
Simplify getAllFilesInPath with single glob call

### PR DESCRIPTION
**Pull request by mishushakov**
## Summary
Simplified `getAllFilesInPath` in both JS and Python SDKs by using a single glob call with multiple patterns instead of nested sequential calls. This eliminates the need to loop through matched directories and make additional glob calls for each one.

## Changes
- Changed from `glob(src)` + loop + `glob(dir/**/*)` to `glob([src, src/**/*])`
- Reduces complexity and improves performance by avoiding multiple filesystem traversals
- All 25 tests pass (12 JS + 13 Python)

🤖 Generated with [Claude Code](https://claude.com/claude-code)

---

> [!NOTE]
> **Low Risk**
> Small refactor to file globbing logic; risk is limited to edge-case differences in which paths are returned for certain patterns/directories.
> 
> **Overview**
> Simplifies Python SDK template file discovery by replacing per-directory recursive globbing with a single `glob.glob([src, src/**/*])` call, normalizing paths for cross-platform matching, and deduplicating results.
> 
> Adds a changeset to publish patch releases for `@e2b/python-sdk` and `e2b` reflecting this internal behavior/performance change; directory inclusion is now handled via a simple filter when `include_directories` is false.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit 2bdbb4175e7758caa50768d451e207e52125a2c7. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>

### CODE REVIEW

File: packages/python-sdk/e2b/template/utils.py [hunk 1]
```diff
@@ -81,36 +81,27 @@ def get_all_files_in_path(
...
+    normalized_src = normalize_path(src)
+
+    # Match both the pattern and its recursive contents in one call
+    # This handles directories (src -> src + src/**/*) and file patterns (*.txt -> just files)
+    patterns = [normalized_src, f"{normalized_src}/**/*"]
+
     files_glob = glob.glob(
-        src,
+        patterns,
         flags=glob.GLOBSTAR,
```
[the remark this answers is not in the document]
Comment (mishushakov) [reply packages/python-sdk/e2b/template/utils.py:93]: it's not Python's glob.glob, it's wcmatch
